Examples/echo/README.md

```diff
@@ -12,27 +12,32 @@ the four RPC types.
 The tool uses the [SwiftNIO](https://github.com/grpc/grpc-swift-nio-transport)
 HTTP/2 transport.
 
+## Prerequisites
+
+You must have the Protocol Buffers compiler (`protoc`) installed. You can find
+the instructions for doing this in the [gRPC Swift Protobuf documentation][0].
+
 ## Usage
 
 Build and run the server using the CLI:
 
 ```console
-$ swift run echo serve
+$ PROTOC_PATH=$(which protoc) swift run echo serve
```

Comment:
Similar to the comment I left on the examples, should we mention (and/or link to the plugin docs) why setting this environment variable is required?

Reply:
Yup, done!